.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         name: '${{ env.CACHIX_NAME }}'
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
         authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-    - run: nix --experimental-features 'nix-command flakes' flake check -L
+    - run: nix --experimental-features 'nix-command flakes' --option allow-import-from-derivation true flake check -L
```

Comment:
This can be removed when https://github.com/NixOS/nixpkgs/pull/274077 is merged, just a hack to get tests passing.